shoop/admin/modules/products/views/edit.py

```diff
@@ -102,11 +105,48 @@ def form_valid(self, form):
             form.forms["attributes"].save()
 
 
+class BaseProductMediaFormPart(FormPart):
+    def get_form_defs(self):
+        yield TemplatedFormDef(
+            self.name,
+            self.formset,
+            template_name="shoop/admin/products/_edit_media_form.jinja",
+            required=False,
+            kwargs={"product": self.object, "languages": settings.LANGUAGES}
+        )
+
+
+class ProductMediaFormPart(BaseProductMediaFormPart):
+    name = "media"
+    priority = -700
+    formset = ProductMediaFormSet
+
+    def form_valid(self, form):
```

Comment:
Can you hoist this up to `BaseProductMediaFormPart`? Seems to be the same code in `ProductMediaFormPart` and `ProductImageMediaFormPart`.